docs/setup/install.md

```diff
@@ -42,6 +42,19 @@ VERSION=v0.1.0
 kubectl apply -f https://github.com/kubernetes-sigs/jobset/releases/download/$VERSION/prometheus.yaml
 ```
 
+### Use cert manager instead of internal cert
+To use cert-manager instead of internal cert, you first need to install cert-manager on your cluster. To do this, run the following command:
+
+```shell
+VERSION=v1.11.0
```

Comment:
just noticed this, where did you find this command using the VERSION env var? In the docs I see the release v1.11.0 being specified in the URL: https://cert-manager.io/docs/installation

`kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml`

Reply:
refs from https://github.com/kubernetes-sigs/jobset/blob/b671aaa0840232e597b93d5725a953197a9e9813/docs/setup/install.md?plain=1#L30-L32

it's shell command so it will replace the VERSION and use v1.11.0 when you actually run it

Reply:
sorry i missed the `$VERSION` in there when I read this the first time. looks good!